tensorboard/data/provider.py

```diff
@@ -73,6 +73,22 @@ class DataProvider(object):
     for 3 random inputs from this batch". A single blob can of course be
     represented as a blob sequence that always has exactly one element.
 
+    When reading time series, *downsampling* refers to selecting a
+    subset of the points in each time series. Downsampling only occurs
+    across the step axis (rather than, e.g., the blobs in a single blob
+    sequence datum), and occurs individually within each time series.
+    When downsampling, the latest datum should always be included in the
```

Comment:
Can you add a TODO to test this at https://github.com/tensorflow/tensorboard/blob/168a9645d9d129267f65e4468c6177ccba65e30f/tensorboard/backend/event_processing/data_provider_test.py#L512 (and ideally the parallel internal places too)?

Reply:
Sure. Done.